src/diffusers/pipelines/pag/pipeline_pag_sd_xl_inpaint.py

```diff
@@ -837,7 +837,7 @@ def _encode_vae_image(self, image: torch.Tensor, generator: torch.Generator):
 
         return image_latents
 
-    # Copied from diffusers.pipelines.stable_diffusion_xl.pipeline_stable_diffusion_xl_inpaint.StableDiffusionXLInpaintPipeline.prepare_mask_latents
+    # Modified from diffusers.pipelines.stable_diffusion_xl.pipeline_stable_diffusion_xl_inpaint.StableDiffusionXLInpaintPipeline.prepare_mask_latents
```

Comment:
thanks! is it possible to keep this method `Copied from` but modify the output instead? e.g. this is how we modify ip_adapter_image_embeds for PAG https://github.com/huggingface/diffusers/blob/f848febacdc54c351ed0ed23fcc4c9349828021e/src/diffusers/pipelines/pag/pipeline_pag_sd_xl_inpaint.py#L1558C1-L1559C1

it's a little bit weird and requires more code but this way it will be easier for us to maintain all the PAG pipelines

in the future, we should make sure these methods that prepare conditions for the denoiser model always return negative condition and condition instead of a combined output